src/pass.cpp

```
@@ -448,45 +448,67 @@ auto Pass::listKeys(const QString &keystring, bool secret) -> QList<UserInfo> {
+namespace {
+
+bool containsAny(const QString &str, const QStringList &patterns) {
+  for (const QString &p : patterns) {
+    if (str.contains(p)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+bool containsAnyCaseInsensitive(const QString &str,
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)